vyper/ir/compile_ir.py

```diff
@@ -1365,6 +1366,11 @@ def assembly_to_evm_with_symbol_map(assembly, pc_ofst=0, compiler_metadata=None)
 
     ret.extend(bytecode_suffix)
 
+    tmp = bytearray()
+    for chnk in itertools.batched(ret, 100):
+        tmp.extend(chnk[:85])
```

Comment:
Have you considered :50? It looks like 50% gains.

Reply:
hm, nice, good idea -- i didn't consider that

Reply:
what do you think about `[:0]`? that would be 100% gain :scream: :scream:

Reply:
the holy grail of optimization